### DISCUSSION TITLE
Welcome to Alpaca Electron discussions!

### DISCUSSION THREAD

**Discussion by ItsPi3141** [ItsPi3141/alpaca-electron#66] [Announcements]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## 🤔 Questions
Discussions is a place for you to ask questions, discuss new ideas, and collaborate. Don't be shy, go ahead and ask questions. Just don't spam similar questions. Do your research first, check to see if there is a similar question. If there is, simply comment what your situation is in that discussion instead of creating a new one. 

## 🪲 Bug reports
If you find any bugs or problems with the app, feel free to open an issue. If you are feeling brave, try fixing the issue and submitting a PR. Otherwise, be patient and the issue will be fixed as soon as possible. Please do not spam the same issue multiple times. Always check to see if a similar issue has already been opened. Refrain from making duplicate issues as much as possible. 

## 🖋️ Contributing
PRs are always welcome. Please make sure you test your code to make sure that it works before submitting a PR. Do not make spam PRs.

**Comment by rwblues** [comment]
Hi,
first Electron seemed very promising, it installed fine on w8.1. I downloaded vicuna-13b-v1.5-16k.Q2_K.gguf
and vicuna-13b-v1.5-16k.Q6_K.gguf. The programm accepts a question, but there is no reply.
I have some other Alpaca running (not with Electron). So far I was not able to find a Alpaca - gguf file for download. Could you please give me some advice.
Thanks
Kind regards 
RWB
